ui/controls.js

```diff
@@ -1521,6 +1545,14 @@ shaka.ui.Controls = class extends shaka.util.FakeEventTarget {
  * @exportDoc
  */
 
+/**
+ * @event shaka.ui.Controls#PlayerUnloaded
+ * @description Fired after the player unloads
```

Comment:
This is not true. Your "unload" method does not wait for the player to finish unloading before it fires the event (`shaka.Player.unload` is an asynchronous method) so this event is actually fired when the player begins to unload.
So I would re-write this documentation, and probably rename the event to `playerunloading`.

Reply:
Sorry I didn't knew that, renamed the event to `playerunloading` now. Thanks for the review!